news/746-update-helpsupport-page

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* (#746) Updated the Help and Support page to include more up to date information on feedstocks, where to access existing feedstocks, and where to contribute new feedstocks.
```

Comment:
Almost! Let's move the PR# to the end.

```suggestion
* Updated the Help and Support page to include more up-to-date information on feedstocks, where to access existing feedstocks, and where to contribute new feedstocks. (#823)
```

Reply:
Wait, is it supposed to be the PR number and not the issue number? @kenodegard

Reply:
I use PR # as an issue # can be nonexistent or could refer to multiple PRs. A PR # will always exist and be unique for a given change.

Reply:
Gotcha, good to know!